lidar.proto

```diff
@@ -0,0 +1,19 @@
+// Version 1.0 (not enforced)
```

Comment:
```suggestion
// Version 1.0
```

`not enforced` means there is no version check in the message itself, but here there is one. Whether it will be checked in the code right now is a "maybe, but it probably should be"